deploy/helm/scf/assets/operations/instance_groups/diego-api.yaml

```diff
@@ -18,6 +18,9 @@
 - type: replace
   path: /instance_groups/name=diego-api/jobs/name=bbs/properties/diego/bbs/advertisement_base_hostname?
   value: {{ .Values.deployment_name }}-diego-api
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=bbs/properties/diego/bbs/health_addr?
+  value: 0.0.0.0:8890
```

Comment:
We _could_ drop this, and do a local curl instead; but I didn't see the harm in exposing the health check endpoint internally.